### PR TITLE
fix(argo-workflows): Updated chart to reflect change to ARGO_BASE_HREF env

### PR DESCRIPTION
**Pull request by scottmelhop**
Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

**Comment by tico24** [comment]
Thanks for this. Can you please address the changelog in chart.yaml?

**Comment by scottmelhop** [comment]
I'm not sure I understood the instructions for the changelog, do I replace this?
```
  artifacthub.io/changes: |
    - kind: fixed
      description: Remove excessive RBAC privileges from workflow-role.
```

> Thanks for this. Can you please address the changelog in chart.yaml?

File: charts/argo-workflows/Chart.yaml [hunk 1]
```diff
@@ -3,18 +3,18 @@ appVersion: v3.6.0
...
+version: 0.44.1
 icon: https://argo-workflows.readthedocs.io/en/stable/assets/logo.png
 home: https://github.com/argoproj/argo-helm
 sources:
-  - https://github.com/argoproj/argo-workflows
+- https://github.com/argoproj/argo-workflows
 maintainers:
-  - name: argoproj
-    url: https://argoproj.github.io/
+- name: argoproj
```
Comment (tico24) [inline charts/argo-workflows/Chart.yaml:12]: nit: I'd prefer if these didn't change. I suspect your IDE did helping.

Comment (scottmelhop) [reply charts/argo-workflows/Chart.yaml:12]: Yeah sorry

**Review by tico24 — APPROVED** [review]
Thanks!

**Review by yu-croco — APPROVED** [review]

**Comment by agilgur5** [comment]
For reference, this was the change I originally made in #2756 before reverting that in #2770 as this change requires Argo Workflows 3.6.0+

**Pull request merged**